experiments/mlflow.py

```diff
@@ -68,6 +68,36 @@ def _log_experiment(experiment: Experiment, parent_name: str, depth: int) -> dic
                 result = _log_experiment(subexperiment, parent_name=run_name, depth=depth + 1)
                 if result is not None:
                     results.append(result)
+                if "Decisioasdn Tree" in experiment.name:
```

Comment:
why is there a weird typo?

additionally, I don't think that we, theoretically, want this case in the master to make the code more readable, however, might be the more practical choice